go/obscuronode/host/hostrunner/host_runner.go

```diff
@@ -13,6 +14,8 @@ import (
 	"github.com/obscuronet/obscuro-playground/go/obscuronode/host/p2p"
 )
 
+const logPath = "host_logs.txt"
```

Comment:
Should this be in the config ? 
What if the defaults was the .build dir ?

Reply:
Yes to making this part of the config.

I don't want the default to be the .build dir though. The default should be what makes sense when it's a standalone binary being run by someone IMO.

Reply:
Do we need a logs folder ? But maybe that's something for the future.